iffig/Biking_Game
# Uc3 move player

# UC3: Move Player (Isabella Figueroa)

## Description
Player Presses one of four arrow keys to progress through bike course. 

## Actors
Player

## Pre-conditions
Player will press arrow keys to specify various movements.

## Post-conditions
Player's position will be updated, and speed/direction adjusted accordingly. 

## How to test (Flow of Events)
 - Check out this branch
 - Hit the 'play' button in intelliJ
 - After you see the moving square, follow the Flow of Events table:

|   | Actor Action                 | System Response                                                   |
|---|------------------------------|-------------------------------------------------------------------|
| 1 | Player presses the arrow key | System updates position and speed                                        |
| 2 |                              | System renders player's new position on screen |



## Questions/ comments / concerns

- The only way I could figure out how to do this is by making our Bouncing Ball Object public, not private.
- Currently movement works though so its a start
- Todo: Figure out how to move this to another class - and use just KeyListener Concepts (its a little neater)
- We'll also need to change Bouncing Ball Object into "Sprite" class for clarity.
